### ISSUE TITLE
[Public Map] Debug why loading map from CARTO throws an error now

### ISSUE THREAD

**Issue by jesusbotella** [CartoDB/kepler.gl#32]
Kepler throws an error in the console when accessing a map in CARTO after saving it.

**Comment by jesusbotella** [comment]
In the process of debugging this error, I found some behaviours that have some room for improvement.

One of these is the before mentioned in our dailies of SQL tables migration. As I had to update the toolkit version and we added more fields to some tables, it was throwing an error of relation missing because of that. It was easy to solve for me because I could recreate the Custom Storage tables, but the thing is that we were not showing any errors to the user and it seemed broken.

Another error that I haven't solved yet is that if you share the same visualization twice (and the first one failed), one of our SQL API requests fails telling us that the name is taken already. But we don't even show the previous URL, or any messages telling the user to changing the name of the visualization.

So that's what I faced while trying to reproduce/solve the error of loading an existing map to Kepler.gl.
